src/common/midi_event.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass, field
+import mido
+
+
+@dataclass(order=True)
+class MidiEvent:
+    event: mido.Message=field(compare=False) # Don't use this when sorting in a priority queue
+    timestamp: int # TODO: Better time class?
```

Comment:
I think a float makes sense here. The usual way to get timestamps in Python is `time.time()` which returns seconds since epoch. Seconds makes sense as a unit and would transfer well if we ever needed to switch to relative time.

If we stick with an int that's fine but I'd just make it clear it's in milliseconds since the epoch.

Reply:
Sounds good to me